WIP-SQL/types-of-database-servers.md

```diff
@@ -0,0 +1,69 @@
+# Database server types
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[link to learn more](https://enki.com)'
+
+---
+## Content
+
+Depending on the user's needs, we can distinguish different types of database servers.
+
+*MySQL* is one of the world's most popular, open source databases. It is best known for its web application database services. It pays an important role in the so called LAMP[1] web development platform. MySQL supports many different platforms, such as: Microsoft Windows, Linux and Mac OS X.
+
+Some MySQL features:
+- high performance
+- open source freedom (cheapest option)
+- 24 x 7 support
+
+*MS SQL Server* is a RDBMS server developed by Microsoft to store and retrieve data as requested by the user. It's suitable for different workloads and it uses primary query languages such as: T-SQL and ANSI SQL.
+
+Some MS SQL Server features:
+- database mirroring
+- XML integration
+- database mail
+
+*Oracle* is a multi-user database. It aims to manage big data collections among multiple clients requesting and sending data.  It's an excellent database server choice for client-server computing. It supports all major operating systems for both, clients and servers.
+
+Some Oracle features:
+- concurrency
+- resource manager
+- self managing database
+
+*MS Access* it's an entry level database management software. It's a powerful database for small-scale projects. MS Access is the first choice of software developers to develop application software. It uses the Jet database engine to deploy a graphical user interface.
+
+Some MS Access features:
+- option of importing and exporting the data to many formats including Excel, Outlook, ASCII
+- microsoft Access offers parameterized queries
+- microsoft Access is a file server-based database
+
+---
+## Practice
+
+Suppose you want to learn coding and you decided to build your first web app. Which server would you choose for hosting your database?
```

Comment:
Change this question to be about fundamental differences between major databases- the answer to this question isn't a settled fact.